AWS/EC2-Linux/System/telemetry/installer/otel_config.yaml

```diff
@@ -0,0 +1,16 @@
+receivers:
+processors:
```

Comment:
you should have the following processor as well
`  resourcedetection/system:
    detectors: ["system"]
    system:
      hostname_sources: ["os"]`